coveralls.sh

```diff
@@ -27,7 +27,12 @@ all_tests_passed=true
 echo "mode: atomic" > profile.cov
 for dir in ${DIR_SOURCE};
 do
-    go test -short -race -covermode=atomic -coverprofile=$dir/profile.tmp $dir
+    if [[ $dir =~ "./services/" ]]; then
+        go test -v -short -race -covermode=atomic -coverprofile=$dir/profile.tmp $dir
+    else
+        go test -short -race -covermode=atomic -coverprofile=$dir/profile.tmp $dir
+    fi
```

Comment:
it was to try to debug the services data race / timeout stuff but here it does not matter I removed it